### ISSUE TITLE
AttributeList has :item in metatable

### ISSUE THREAD

**Issue by daurnimator** [craigbarnes/lua-gumbo#53]
AttributeList 

``` lua
b=require"gumbo".parse([[<html><body xmlns:foo="bar" foo:lol="qux"></body></html>]]).body
print(b.attributes:item(1)) -->  attempt to call method 'item' (a nil value)
for k,v in pairs(getmetatable(b.attributes)) do print(k,v) end --[[
__index function: 0x886ec0
item    function: 0x886d10
]]
```

**Comment by craigbarnes** [comment]
I think I added the `:item()` method way back when I was blindly implementing methods from the DOM spec. It's currently not included via the `__index` lookup. Rather than add it to the `__index` lookup, I think it's probably better to delete the implementation and just use `b.attributes[1]`.

**Comment by craigbarnes** [comment]
Closed by 5c46e672254c1cf1493971595a9ef018bac21618.
